templates/Dockerfile.template

```diff
@@ -1,85 +1,68 @@
 FROM {{ .From }} as builder
 
 # Update and install necessary packages
-{{- $etcd := index .Packages "etcd" }}
-
-RUN set -ex; \
-	apt-get update && \
-	apt-get -y install \
-	gnupg \
-	wget \
-	golang \
-	govendor
-
-# This is the release of Etcd to pull in.
-ENV ETCD_VER={{ $etcd.Version }}
-ENV ETCD_FILE=etcd-v${ETCD_VER}-linux-amd64.tar.gz
-
-ENV GOPATH /usr/local
-ENV ETCD_SHA256 {{ $etcd.Sha256 }}
-
-# Download etcd version v{{ $etcd.Version }} and put binary files into /usr/local/bin
-RUN set -ex; \
-    # Downloading source file
-    wget -q https://github.com/etcd-io/etcd/releases/download/v${ETCD_VER}/${ETCD_FILE} \
-    # Check SHA256sum
-    && echo "${ETCD_SHA256}" "${ETCD_FILE}" | sha256sum -c - \
-    # Extracting and adding to binary directory
-    && tar xzvf "${ETCD_FILE}" -C /tmp \
-    && cp /tmp/etcd-v${ETCD_VER}-linux-amd64/etcd /usr/local/bin/ \
-    && cp /tmp/etcd-v${ETCD_VER}-linux-amd64/etcdctl /usr/local/bin/
-
-RUN set -eux \
-    # Downloads source code
-    && wget -q https://github.com/etcd-io/etcd/archive/v${ETCD_VER}.tar.gz \
-    && mkdir -p "${GOPATH}/src/github.com/etcd-io/etcd" \
-    && tar -xzf v${ETCD_VER}.tar.gz --strip-components=1 -C "${GOPATH}/src/github.com/etcd-io/etcd"
-
-RUN set -eux \
-    # Extracts licences
-    && cd "${GOPATH}/src/github.com/etcd-io/etcd" \
-    && govendor license +vendor > /NOTICES
+RUN apt-get update && \
+    apt-get install -y wget git tar gzip build-essential
+
+# Install Go
+ARG GO_VERSION=1.20
```

Comment:
add golang to the packages and use the package variables

Reply:
Fixed. Golang was added to the versions.yaml and have been used into the image.